.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: php
 sudo: false
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - vendor
 php:
 - 5.3
 - 5.4
 - 5.5
 - 5.6
+- 7
```

Comment:
`7` errors out, and is required to be `7.0` (now that there is an official release).

This can be removed, as it's been added in a separate PR